Formula/cargo-release.rb

```diff
@@ -0,0 +1,26 @@
+class CargoRelease < Formula
+  desc "Cargo subcommand `release`: everything about releasing a rust crate"
+  homepage "https://github.com/crate-ci/cargo-release"
+  url "https://github.com/crate-ci/cargo-release/archive/refs/tags/v0.24.4.tar.gz"
+  sha256 "7645ced715c7629fcef39e786fa89c946183f1adc84d28df524605bc92988d5f"
+  license any_of: ["Apache-2.0", "MIT"]
+  head "https://github.com/crate-ci/cargo-release.git", branch: "master"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
```

Comment:
```suggestion
```
Doesn't look necessary. Does `brew livecheck cargo-release` not work if you leave this out?

Reply:
I was copying it from other `cargo-` formulas. it indeed isn't required, I've removed it